graphql/src/adapters/webserver/run.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
I don't know why Github doesn't remember that I did a proper `git -mv` command, but this used to be webserver/main.py